text/0008--individual-agency-company-pages.md

```diff
@@ -0,0 +1,131 @@
+- Start Date: 2018-07-24
+- RFC PR: 
+- Gatsby Issue: 
+
+# Summary
+
+As a follow-up to the [Site Showcase](https://next.gatsbyjs.org/showcase/), we want to create individual & organization pages highlighting the individual community members and what they've built. 
+
+We also want to create a free Job Board highlighting opportunities for employment building Gatsby websites at agencies and companies using Gatsby!
+
+# Motivation
+
```

Comment:
It'd provide context to hear more about what you have heard people wanting, i.e. (this is stuff I've written down from what I heard you and Kyle say)

- People are wanting to hire programmers who are skilled at using Gatsby already
- People are wanting to have someone build a Gatsby site for them
- Many individuals/agencies/companies are producing so much valuable OSS work and many beautiful websites and it'd be useful for them to market themselves to the audience that already wants them

Reply:
done!